File: 09 BMI/BMI.py
#учитавање тежине и висине
k = float(input())
h = float(input())

#израчунавање BMI
BMI = k / (h * h)

if(BMI < 18.5):
	print ("NEUHRANJENOST")
elif(18.5 <= BMI < 30):
	print ("IDEALNA MASA")
else:
	print ("GOJAZNOST")

File: 09 BMI/BMI.cpp
#include <iostream>

using namespace std;

int main(){

	double k,h,BMI;
	
	 //учитавање тежине и висине
	cin >> k;
	cin >> h;

	 //израчунавање BMI
	BMI = k / (h * h);

	if(BMI < 18.5)
		cout << "NEUHRANJENOST" << endl;
	else if(18.5 <= BMI && BMI < 30)
		cout << "IDEALNA MASA" << endl;
	else
		cout << "GOJAZNOST" << endl;
	 
	return 0;
}